docs/acceptance.md

```diff
@@ -0,0 +1,42 @@
+<img width="840px" height="auto" src="./acceptance-wires.png" alt="Splitting basics">
+
+### In General
+- [ ] It should generally match Photon styles.
+- [ ] It should have a way for users to provide feedback without leaving the experiment.
+- [ ] It should match final visual specs.
+
+### Context Clicks
+- [ ] It should register a _link_ context menu item for opening sites in the sidebar
+- [ ] It should register a _page_ and _tab_ context menu item for opening sites in the sidebar
```

Comment:
#4 (implemented). Note there's also #16, which is to add a context menu to bookmarks (should be simple)